cmd/snap/cmd_man.go

```diff
@@ -41,6 +41,11 @@ func init() {
 
 func (*cmdMan) Execute([]string) error {
```

Comment:
The longManHelp and shortManHelp might also be updated above after the conventions we agreed to in snappy-internal or snappy-devel (I forget which).